atlasclient/client.py

```diff
@@ -218,7 +220,10 @@ def request(self, method, path, **params):
         return self.handle_response(method, response)
 
     def construct_resource_url(self, path):
-        url_template = "{base_url}/v{version}/{resource_path}"
+        if path and path[0] == '/':
+            url_template = 'https://cloud-dev.mongodb.com{resource_path}'
```

Comment:
We should use the `base_url` config var to set this instead of hardcoding it... Does the cloud-dev API not use the same convention as the public API?

Reply:
I have opened #99 that makes this change unnecessary.